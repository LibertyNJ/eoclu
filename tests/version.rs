//! Integration tests that exercise version functionality.

use util::assert_output_snapshot;

mod util;

#[test]
fn should_display_version_when_passed_long_flag() {
    let mut cmd = util::create_test_command();
    let assert = cmd.arg("--version").assert().success().code(0);
    assert_output_snapshot!(assert.get_output());
}

#[test]
fn should_display_version_when_passed_short_flag() {
    let mut cmd = util::create_test_command();
    let assert = cmd.arg("-V").assert().success().code(0);
    assert_output_snapshot!(assert.get_output());
}
