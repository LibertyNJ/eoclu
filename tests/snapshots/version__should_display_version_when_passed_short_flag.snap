---
source: tests/version.rs
expression: snapshot
---
stdout:
eoclu 0.1.0

stderr:
